packages/suite/src/components/suite/TransactionsGraph/components/SkeletonTransactionsGraph.tsx

```diff
@@ -17,11 +17,11 @@ const SkeletonWrapper = styled.div`
     }
 `;
 
-const SkeletonBar = (props: React.ComponentProps<typeof SkeletonRectangle>) => (
+export const SkeletonBar = (props: React.ComponentProps<typeof SkeletonRectangle>) => (
```

Comment:
It wasn't exported and it's not used anywhere else

Reply:
done